README.md

```diff
@@ -133,7 +133,7 @@ on the terminal run
 
 after the last line add
 
-`@lxterminal -e sh /home/pi/PVM/launch.sh`
+`@lxterminal -e sh $HOME/PVM/launch.sh`
```

Comment:
I think you also need to change the path in [launch.sh](https://github.com/omarcostahamido/PVM/blob/main/launch.sh)

Reply:
Good catch!